README.md

```diff
@@ -1,13 +1,93 @@
+Mapping states, counties, and municipalities that are enacting emergency tenant protections due to COVID-19 (the novel corona virus) pandemic, as well as where organized rent strikes are taking place.
 
 ## Data Source
 
-Data sourcing is being provided by the [Anti-Eviction Mapping Project](https://www.antievictionmap.com/).
+Data sourcing and maintenance is being provided by the [Anti-Eviction Mapping Project](https://www.antievictionmap.com/).
 
-## Running locally
+**DISCLAIMER:** This data is by no means perfect or exhaustive of all emergency tenant protection policies in the United States and elsewhere. It has been crowdsourced and is maintained by a team of dedicated volunteers. If you notice something missing or incorrect in the data, please [reach out to us](mailto:antievictionmap@riseup.net) to let us know so we may update it accordingly!
 
-Run a simple server, e.g. `python -m SimpleHTTPServer` if you have Python.
+## Developer Instructions
 
-Then, in your browser, go to the port your server runs on, which is usually port 8000, so visit http://localhost:8000/
+Getting this project running locally requires that:
+
+1. You are comfortable running programs on the [CLI](https://en.wikipedia.org/wiki/Command-line_interface) such as the [Terminal](https://support.apple.com/guide/terminal/welcome/mac) program on MacOS.
+
+2. You have installed [NodeJS](https://nodejs.org/en/) >= `v10.13` and either the [Yarn](https://yarnpkg.com/) >= `1.22` or [NPM](https://www.npmjs.com/) >= `6.4` (NPM should automatically be installed with NodeJS). Older versions of any of these may or may not work.
+
+### Develop
+
+First, in the root level of this repo install the required package dependencies by doing:
+
+```
+yarn install
+```
+
+or
+
+```
+npm install
+```
+
+To start a local web server with live reload do:
+
+```
+yarn start
+```
+
+or
+
+```
+npm start
+```
+
+Then visit `localhost:8080` in your browser.
+
+---
+
+To create a production optimized build that will be outputted in the `dist` directory do:
+
+```
+yarn build
+```
+
+or
+
+```
+npm run build
+```
+
+### Deploying
+
+To deploy the site to Github Pages on the `gh-pages` branch (this will also run the `build` script above prior to publishing) do:
+
+```
+yarn deploy
+```
+
+or
+
+```
+npm run deploy
+```
+
+You will need to have write privileges to this repository on Github to be able to do this.
+
+**NOTE: Use caution when doing this**, *before deploying you should make sure your build is successful and runs as expected. You may do this by running another webserver, for example using Python, in the `dist` directory after a build, and then viewing the site in your browser:*
+
+``` bash
+# first create a production build
+yarn build
+
+# assuming the build was successful,
+# change directories to the output directory:
+cd dist
+
+# run a local server using python and view your changes on localhost:8000
+python -m SimpleHTTPServer 8000
```

Comment:
this is a small thing, but it's possible to use `http-server` when installed via `npm i -g http-server`, which is nice because it has more features like hot reloading and it's in node.

Reply:
@river-honer we could also include `http-server` as a dev dependency and have a script for `start:prod` which would run the `build` script then run `http-server` on the `dist` directory. Might make it easier for folks who are newer to frontend and build systems?

Reply:
Added an `npm` script for this & updated this part of the README